data/campaigns/World_Conquest/scenarios/WC_II_scenario.cfg

```diff
@@ -0,0 +1,89 @@
+## macros may not be used inside translatable strings.
+
+#define WC_II_CAMPAIGN_NAME_1P
+_ "World Conquest 1p" #enddef
+
+#define WC_II_CAMPAIGN_NAME_2P
+_ "World Conquest 2p" #enddef
+
+#define WC_II_CAMPAIGN_NAME_3P
+_ "World Conquest 3p" #enddef
+
+#define WC_II_CAMPAIGN_DESC_1P
+	_ "A randomly generated campaign for 1 player. It has 6 level of difficulties.
+(Expert level, 5 scenarios.)" #enddef
+
+#define WC_II_CAMPAIGN_DESC_2P
+	_ "A randomly generated cooperative campaign for 2 players. It has 6 level of difficulties.
+(Expert level, 5 scenarios.)" #enddef
+
+#define WC_II_CAMPAIGN_DESC_3P
+	_ "A randomly generated cooperative campaign for 3 players. It has 6 level of difficulties.
+(Expert level, 5 scenarios.)" #enddef
+
+
+#define WC2_CAMPAIGN_NEW PLAYERS
+## Everything that usually goes into [scenario] like [event], [load_ressource], map_data 
+## etc is generated by lua. This is also why there is only one [multiplayer] tag: the next
+## scenario is again "WC_II_{PLAYERS}p" and the lua code detects that is was called a second
+## time and generated the second scenario. (same for third, fourth and fifth). All the lua
+## code inside ./lua/map does map/scenario generation.
+[multiplayer]
+	id = "WC_II_{PLAYERS}p{ID_SUFFIX}"
+	name= {WC_II_CAMPAIGN_NAME_{PLAYERS}P} + {ID_SUFFIX}
```

Comment:
The description is not correctly display. 
From what I have tested, it should be `name= {WC_II_CAMPAIGN_NAME_{PLAYERS}P} + "{ID_SUFFIX}"`.

Reply:
oh yes probably, i just tested it with a  nonempty ID_SUFFIX yet.